server/tests/signin_tests.js

```diff
@@ -0,0 +1,69 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import chaiThings from 'chai-things';
+import { before } from 'mocha';
+import FakeUser from '../mock/FakeUser';
+import app from '../app';
+
+chai.use(chaiHttp);
+chai.use(chaiThings);
+const { expect } = chai;
+
+const user = new FakeUser();
+const userCredentials = user.generateFakeUser();
+
+describe('Test POST /api/v1/auth/signin/', () => {
+  let signinData = {};
+  before((done) => {
+    chai.request(app)
+      .post('/api/v1/auth/signup')
```

Comment:
@T2Wil  Kindly update your APIs to `/api/v2`

Reply:
@mcaleb808, Solved!